youtube_dl/extractor/crunchyroll.py

```diff
@@ -338,6 +338,9 @@ def _real_extract(self, url):
                 'height': int_or_none(xpath_text(metadata, './height')),
                 'width': int_or_none(xpath_text(metadata, './width')),
             }
+            
+            if video_url is None:
```

Comment:
This check should take place earlier and should be more general `if not video_url`. Same should be done for `video_play_path`.